components/input/test/auro-input.test.js

```diff
@@ -67,19 +67,8 @@ describe('auro-input', () => {
     `);
 
     const clearButton = el.shadowRoot.querySelector('.clearBtn');
-    const input = el.shadowRoot.querySelector('input');
```

Comment:
Why were these parts removed from the test?

Reply:
I'm guessing because it was not being used?

Reply:
this file is just copied from the latest `auro-input`